tests/Feature/Sensors/JobAttemptSensorTest.php

```diff
@@ -30,17 +31,30 @@
 
     setTraceId('0d3ca349-e222-4982-ac23-2343692de258');
     Config::set('queue.default', 'database');
+    Redis::command('FLUSHALL');
 })->skip(version_compare(Application::VERSION, '11.0.0', '<'), 'Laravel 10 support is pending');
 
-it('ingests processed job attempts', function () {
+$workCommands = [
+    'queue:work',
+    'horizon:work',
+];
+
+$workOptions = [
+    '--max-jobs' => 1,
+    '--sleep' => 0,
+    '--stop-when-empty' => true,
+    '--tries' => 1,
+];
```

Comment:
In order for the tests to pass, we run the `horizon:work` command.

Usually in your application, you would run the `horizon` command which then passes through the `--tries=1` option.

When running the `horizon:work` command, `--tries` defaults to `0`. This meant that our failed job test was not seeing the job fail.

To fix this I added the `--tries=1` to standardise both of the commands and how they work in the real-world (`queue:work` defaults to `1`).

While doing this I decided to extract the sensible defaults here and then just augment any options in place as needed.

Reply:
Self-resolving to merge after approvals.